README.md

```diff
@@ -156,33 +203,27 @@ Define and configure SKALogger TANGO Device server as specified in the given scr
 * [Logging Configuration](https://drive.google.com/file/d/1ExPvqqKg-2tXPZg_-rQophfDV5LpTa-a/view?usp=sharing)
 
 
-### 3.5: Running the GUI
-
-Once all the TMC TANGO devices are up and running, one can proceed to run the taurus GUI by following these steps:
-
-* Navigate to the GUI folder on terminal
-* Run the mainWindow_SKA.py file using the command:
-
-    `python2 mainWindow_SKA.py`
+### 3.5: GUI
 
+The GUI is an optional component not contained in the TMC prototype repository. The GUI is created using Webjive and configured for the TMC devices.
```

Comment:
Can add a link to the WebJive project inside SKA Github organization for reference.

Reply:
Yes. It is added.

Reply:
Commit ID: [4fe0b933c9bfe4e5c66c31e0dfae0261bf485831]